### PR TITLE
Fixes #383

### LINKED ISSUE #383: W001 false positive when using Content-Type meta header
my webpage has:
`<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />`

but I get a W001 error running bootlint on it
I think that a false positive

### PR DESCRIPTION
**Pull request by gemal**

### CODE REVIEW

**Comment by cvrebert** [comment]
So, a few things:
* Lowercase `content-type` ought to be allowed
* Uppercase `UTF-8` ought to be allowed
* Tests need to be added

**Comment by cvrebert** [comment]
https://github.com/twbs/bootlint/tree/master/test describes how to add tests

**Comment by gemal** [comment]
@cvrebert does it look correct?

File: test/bootlint_test.js [hunk 1]
```diff
@@ -75,6 +75,9 @@ exports.bootlint = {
         test.deepEqual(lintHtml(utf8Fixture('charset/utf8.html')),
             [],
             'should not complain when UTF-8 charset <meta> tag is present.');
+        test.deepEqual(lintHtml(utf8Fixture('charset/utf8-meta.html')),
```
Comment (cvrebert) [inline test/bootlint_test.js]: You need to increment the 3 to a 4 on line 74. (See Travis failure)

**Comment by cvrebert** [comment]
Other than that, LGTM.

**Comment by gemal** [comment]
@cvrebert test passed

**Comment by cvrebert** [comment]
Thank you.

**Pull request merged**